### PR TITLE
Added welcome command, caption color improvements and other

### PR DESCRIPTION
**Pull request by dimonDDL**
Added welcome command that is persistent and trolls pyrobot.
Added colors that are human readable for the caption command.
Some other stuff I can't remember.

### CODE REVIEW

File: thatkitebot/cogs/listenercog.py [hunk 1]
```diff
@@ -20,18 +20,22 @@
 
 
 import discord
+from discord import member
```
Comment (ThatRedKite) [inline thatkitebot/cogs/listenercog.py:23]: why?

File: thatkitebot/cogs/welcomecog.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# ------------------------------------------------------------------------------
+#  MIT License
+#
+#  Copyright (c) 2019-2021 dimonDDL
```
Comment (ThatRedKite) [inline thatkitebot/cogs/welcomecog.py:4]: uh sir you are now claiming ownership of the entire bot? Because this cog has not existed in 2019

**Review by ThatRedKite — CHANGES_REQUESTED** [review]
Fix the copyright info smh

File: thatkitebot/cogs/welcomecog.py (hunk 2 above)
Comment (dimonDDL) [reply thatkitebot/cogs/welcomecog.py:4]: Mm. I claim ownership of the file so let me update the date.

File: thatkitebot/cogs/imagecog.py [hunk 3]
```diff
@@ -114,6 +114,11 @@ def reduce(buf, fn):
 
 
 def caption(blob, fn, ct, path):
+    color_alias = {
+        "piss":"#f9fc12",
+        "cum":"#ededd5",
+        "pickle":"#12a612"
+    }
```
[the remark this answers is not in the document]
Comment (dimonDDL) [reply thatkitebot/cogs/imagecog.py:121]: dunno and pup proof

File: thatkitebot/__main__.py [hunk 4]
```diff
@@ -95,6 +96,7 @@ def __init__(self, command_prefix, dirname, tt, help_command=None, description=N
         self.loop.run_until_complete(self.aiohttp_start())
         self.redis = aioredis.Redis(host="redis", db=1, decode_responses=True)
         self.redis_cache = aioredis.Redis(host="redis_cache", db=0, decode_responses=True)
+        self.redis_welcomes = aioredis.Redis(host="redis", db=3, decode_responses=True)
```
[the remark this answers is not in the document]
Comment (dimonDDL) [reply thatkitebot/__main__.py:99]: Under the `db=2` would be ideal.

[the remark this answers is not in the document]
Comment (ThatRedKite) [reply thatkitebot/__main__.py:98]: yeah that will be added soon

**Review by ThatRedKite — APPROVED** [review]

**Pull request merged**